scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/authentication/ManageUsers.java

```diff
@@ -527,17 +683,11 @@ public String toString() {
     static class UserInfo {
         private String login;
         private String password;
-        private String[] groups;
+        private Collection<String> groups = Collections.EMPTY_LIST;
```

Comment:
Not really the same, it handles generics. You should see an unchecked assignment warning if `Collections.EMPTY_LIST` is used.